components/profileHandler.ts

```diff
@@ -829,8 +829,21 @@ async function loadSession(
     sessionData?: ContractSession,
 ): Promise<void> {
     if (!sessionData) {
-        sessionData = await getContractSession(token + "_" + sessionId)
+        try {
+            //First, try the loading the session from the filesystem.
+            sessionData = await getContractSession(token + "_" + sessionId)
+        } catch (e) {
+            //Otherwise, see if we still have this session in memory.
+            //This may be the currently active session, but we need a fallback of some sorts in case a player disconnected.
+            if (contractSessions.has(sessionId)) {
+                sessionData = contractSessions.get(sessionId)
+            } else {
+                //Retrow the error
```

Comment:
I'm being a pedant but, typo.

Reply:
Haha, oops. Totally understand. I'll force push it.